Bundle/CoreBundle/Route/RouteLoader.php

```diff
@@ -41,6 +41,7 @@ protected function addVictoireRouting(RouteCollection &$collection)
             '@VictoireMediaBundle/Controller/',
             '@VictoirePageBundle/Controller/',
             '@VictoireCoreBundle/Controller/',
+            '@VictoireConfigBundle/Controller/',
```

Comment:
This seems to be the only thing that breaks if ConfigBundle is not declared in the Kernel. Add a little test around it would make ConfigBundle optionnal so it worth the cost 😀